IRremoteInt.h

```diff
@@ -211,6 +211,7 @@ typedef struct {
   unsigned int timer;     // state timer, counts 50uS ticks.
   unsigned int rawbuf[RAWBUF]; // raw data
   uint8_t rawlen;         // counter of entries in rawbuf
```

Comment:
Ahh. This explains why I've had problems with codes longer than 256. We should make this an unsigned int.
Please change that while we are here.